src/k8s/pkg/k8sd/app/hooks_bootstrap.go

```diff
@@ -291,6 +291,8 @@ func (a *App) onBootstrapControlPlane(ctx context.Context, s state.State, bootst
 		localhostAddress = "127.0.0.1"
 	}
 
+	cfg.Network.LocalhostAddress = utils.Pointer(localhostAddress)
```

Comment:
This will be shared across all nodes. I just thought about if this can cause any kind of race issues in dualstack environments? e.g. if one node sets `--address ipv6` and one node sets `--address myipv4`? 
I don't think this will be an issue - just a thought to consider before merging this.

Reply:
Well the apiserver listens to any interface and I've changed apiserver-proxy to do the same. For both cases if there is an ipv6 node, all other nodes needs to have ipv6 routing the same way for ipv4 so I believe we won't face issues in that regard. The only issue would be the old behavior on the apiserver-proxy where we gave it a certain local ip to listen to, which might've caused problems. 

I'm not sure if apiserver-proxy listening to any interface has negative effects/complications.